docs/README.md

```diff
@@ -1,115 +1,188 @@
-- [ Media Communications Mesh](#-media-communications-mesh)
-  - [Table of Contents](#table-of-contents)
-  - [Introduction](#introduction)
-  - [Media Proxy](#media-proxy)
-    - [Key Features](#key-features)
-    - [MemIF](#memif)
-  - [MCM SDK](#mcm-sdk)
-  - [Getting Started](#getting-started)
-    - [Prerequisites](#prerequisites)
-    - [Basic Installation](#basic-installation)
-  - [Usage](#usage)
-  - [Known Issues](#known-issues)
-  - [Support](#support)
-  - [Note](#note)
+![Media-Communication-Mesh](./_static/media-communications-mesh-proto-1.webp)
 
 ## Introduction
 
 The Media Communications Mesh (MCM) enables efficient, low-latency media transport for media microservices for Edge, Edge-to-Cloud, and both private and public Cloud environments. The framework creates a secure, standards-based media data plane for inter-microservices communications using a new media proxy leveraging the [Media Transport Library (MTL)](https://github.com/OpenVisualCloud/Media-Transport-Library) and adds the necessary microservices control-plane communications infrastructure to implement any media control protocol.
 
-## Media Proxy
 
-The Media Proxy component in this project is responsible for transporting audio and video stream media data. It handles the routing and forwarding of media data between mesh nodes, ensuring low latency and efficient resource utilization.
+## 1. Media Communications Mesh:
+
+- Framework Introduction: MCM is a framework designed for low-latency media transport. It caters specifically to Edge, Edge-to-Cloud, and Cloud environments, connecting media microservices.
+- Purpose: Its main role is to establish a secure media data plane for inter-microservices communication. This ensures media can be reliably transmitted across different cloud environments.
+- Technological Basis: The Media Transport Library (MTL) helps form the media data plane, and a control-plane communications infrastructure adds support for media control protocols.
 
-It serves as a Data Plane component in a media application's Service Mesh. Media Proxy provides Shared Memory APIs to abstract the complexity of media transport.
+✅ Key point: The main goal is to enable secure, fast, and standardized media communication between microservices, regardless of the environment (Edge or Cloud).
 
-More information about the Media Proxy component can be found in the [media-proxy](media-proxy) subdirectory.
+## 2. Media Proxy Introduction:
 
-### Key Features
+- Core Functionality: The Media Proxy handles the routing and forwarding of media data, specifically audio and video streams, between mesh nodes.
+- Low Latency & Efficiency: The focus is on ensuring low-latency and efficient usage of system resources, which is critical for real-time media applications.
+- Service Mesh Role: It acts as a Data Plane component within a larger service mesh for media applications, utilizing Shared Memory APIs to abstract the complexities involved in media transport.
```

Comment:
I don't understand: "utilizing Shared Memory APIs to abstract the complexities involved in media transport."

I read it as: "media proxy uses shared memory to send media" Which is just one of the options. If you meant node to proxy media transport please specify it.  If you meant proxy to proxy communication please mention at least ST2110

Reply:
Fixed.